search-books/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>nix.education</groupId>
+    <artifactId>search-books</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>search-books</name>
+    <description>search-books</description>
```

Comment:
Do wee need to specify a parent in this pom file?

Reply:
Refactored.